ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -713,7 +709,7 @@ public void shouldFailForMisSpelledTerminateClusterParameters() {
     final KsqlRequest ksqlRequest = new KsqlRequest(TerminateCluster.TERMINATE_CLUSTER_STATEMENT_TEXT, properties);
     final KsqlResource testResource = TestKsqlResourceUtil.get(ksqlConfig, ksqlEngine);
     final Response response = testResource.terminateCluster(ksqlRequest);
-    assertThat(response.getEntity().toString(), equalTo("Invalid SOURCES_LIST_TYPE : KEEEP. SOURCES_LIST_TYPE can either be KEEP or DELETE.\n"));
+    assertThat(response.getStatus(), equalTo(200));
```

Comment:
name of test suggests negative test, but you're checking for 200 status, which is success.